apps/api/src/admin/participant_manager.py

```diff
@@ -14,14 +15,25 @@ class Participant(UserRecord):
 
     first_name: str
     last_name: str
-    status: Status
+    status: Union[Status, Literal[Decision.WAITLISTED]]
 
 
-async def get_attending_applicants() -> list[Participant]:
-    """Fetch all applicants who have a status of ATTENDING"""
+async def get_hackers() -> list[Participant]:
+    """Fetch all applicants who have a status of ATTENDING, WAIVER_SIGNED, CONFIRMED,
+    or WAITLISTED."""
     records: list[dict[str, Any]] = await mongodb_handler.retrieve(
         Collection.USERS,
-        {"role": Role.APPLICANT, "status": Status.ATTENDING},
+        {
+            "role": Role.APPLICANT,
+            "status": {
+                "$in": [
+                    Status.ATTENDING,
+                    Status.WAIVER_SIGNED,
+                    Status.CONFIRMED,
+                    Decision.WAITLISTED,
```

Comment:
Issue: during the walk-in promotion process, there will be a slight moment when the user's status will be `Decision.ACCEPTED`, so we should account for that to show in the list of participants.